data/projects.json

```diff
@@ -77,42 +65,66 @@
     "tags":["LIS","digitization","open source","lib-re","guide","roadmap"],
     "description_short":"Roadmap, guide, and general resource hub for use in the digitization and description of printed materials.",
     "description_long":"This repository seeks to provide a guiding philosphy of use for and between lib-re and other LIS/LAM programs and tools while establishing and managing expectations for lib-re development and support. Its purposes lie primarily in the coordination of activity and the establishment of a contiguous digitization workflow connecting the core steps of the digitization process with the tools best suited for their optimal completion.",
-    "importance":"The creation of dispirate tools without coordinating effort and automation between them hampers their aggregate usefulness. This project exists to coordinate and inform the interfaces between tools and steps, in order minimize this confusion and provide an end-to-end description and guide to the digitization process.",
+    "importance":"The creation of disparate tools without coordinating effort and automation between them hampers their aggregate usefulness. This project exists to coordinate and inform the interfaces between these tools and steps, in order minimize confusion and provide an end-to-end pipeline for the digitization process.",
     "involvement":"creator, contributor, project manager, requirements management"
   },
   {
-    "name":"CommunityCareHub",
+    "name":"crosswalk",
+    "url_web":"https://github.com/lib-re/crosswalk/wiki",
+    "url_src":"https://github.com/lib-re/crosswalk/",
     "photo":"",
-    "url_web":"https://community-care-hub.herokuapp.com/caregiverHome.php",
-    "url_src":"https://github.com/amk5401/CommunityCareHub",
-    "languages":["html","css","php","angular","js"],
-    "tags":["UI","web","design","bootstrap"],
-    "description_short":"Web application made to assist caregivers, patients, and loved ones in an assisted living community. (UI Prototype)",
-    "description_long":"CommunityCareHub is a responsive web application ",
-    "importance":"Designing, developing, and testing this application was a great learning experience in the complexity and was one of my favorite experiences with working in a team.",
-    "involvement":"document, design, develop, manage requirements, test, report"
+    "languages":[],
+    "tags":["LIS","cataloguing","open source","lib-re","metadata"],
```

Comment:
add `MUSE-359` and `school` tags to denote its connection as a project that started with a school project.

Reply:
same with `FeedReader` below as well.